### ISSUE TITLE
Tailscale can not log in in Windows

### ISSUE THREAD

**Issue by Tom-Teamo** [tailscale/tailscale#13073]
### What is the issue?

II have a trouble in Windows11 when using tailscale. I can not use the tailscale after just one night. When I click the icon it shows "Please Restart Tailscale Windows service" and I do restart it but not work at all. I follow the [issure 2282](https://github.com/tailscale/tailscale/issues/2282) and I delete some folder and reinstall tailscale many times still failed. I also tried to remove my device but failed again.

### Steps to reproduce

I do not know, just after one night Tailscale lose connection

### Are there any recent changes that introduced the issue?

Maybe my VPN is offline in the night

### OS

Windows

### OS version

23H2

### Tailscale version

1.70.0

**Comment by Tom-Teamo** [comment]
of course. it's `BUG-3cd8a7737c556cba5f3f057480d11b218df9171351f80b58b378249dee645352-20240809131746Z-424239ff04e3aeed`.

**Comment by Tom-Teamo** [comment]
I do not know why! Just few seconds ago It automatically works! And I did nothing before. If you need more information, pleases let me know. Otherwise, you can close this issure. Thank :)

**Comment by dblohm7** [comment]
close but look at bug report first
